store/storage/store.go

```diff
@@ -143,7 +146,7 @@ func (ss *StorageStore) Restore(version uint64, chStorage <-chan *store.KVPair)
 		}
 	}
 
-	return ss.db.SetLatestVersion(version)
+	return nil
```

Comment:
Asking to learn: Why this change? Do we not want to call `SetLatestVersion`?

Reply:
I realized the latest version is stored while batch writing